source/configuration/write.txt

```diff
@@ -130,6 +130,15 @@ The following options for writing to MongoDB are available:
          guide on the
          :manual:`WriteConcern wtimeout option </reference/write-concern/#wtimeout>`.
 
+   * - ``upsertDocument``
+     - | When true, ``replace`` and ``update`` operations will insert the data
```

Comment:
Suggestion:
1) I'm not sure if there's a standard for formatting values and operations, but it seems confusing when "true" is unformatted whereas in other instances it is formatted with double backticks. 

2) I think the "replace" and "update" operations should be unformatted here based on how, when they are used to describe operations and not setting values, they are unformatted in other parts of this table.